Vagrantfile

```diff
@@ -148,31 +50,18 @@ Vagrant.configure(2) do |config|
   end
 
   if Vagrant.has_plugin?("vagrant-cachier")
-    # The vagrant-cachier plugin (optional) will speed up rebuilds by reusing
-    # downloaded artifacts; to benefit you need to install the plugin on your
-    # host macine by running: vagrant plugin install vagrant-cachier
-    # For more details see: https://github.com/fgrehm/vagrant-cachier
+    # The vagrant-cachier plugin (optional) will speed up rebuilds by reusing downloaded artifacts
     # Configure cached packages to be shared between instances of the same base box.
     config.cache.scope = :box
   end
 
-  # bash history folder needs creating before it is set in the provisioning script below
-  if $windows
```

Comment:
The external bash history folder config feels too custom to roll out to the wider world. Our env shouldn't be creating extra directories really. Maybe could be introduced and moved somewhere more appropriate. Or at the very least mentioned in the README.